catala/cantar/tastar/tastes_sol/tastes_sol.md

```diff
@@ -1,3 +1,7 @@
 T'acostes a l'horitzó i tastes el caramel que cau del sol.
 Té un gust deliciós i sorprenent, com corrents de calor pessigollant entre les teves dents.
-Quan te n'has fartat t'adones que rere l'horitzó no hi ha més caramel: només una llarga caiguda al buit.
\ No newline at end of file
+Quan te n'has fartat t'adones que rere l'horitzó no hi ha més caramel: només una llarga caiguda al buit.
+
+[Estàs tan tip de caramel, que decideixes deixar-te caure al buit. Fins que xoques contra el fons...](../../../aixecar/aixecar.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.